baseplate/server/__init__.py

```diff
@@ -174,6 +177,12 @@ def configure_logging(config: Configuration, debug: bool) -> None:
     root_logger.setLevel(logging_level)
     root_logger.addHandler(handler)
 
+    # add PID 1 stdout logging if we're containerized and not running under init system
+    if _is_containerized() and os.getppid() != 1:
```

Comment:
In my testing, this doesn't seem to cause double logging. But I don't understand why. If we're running directly under dumb-init, we should have inherited stdout from that parent process and so writing to stdout is the same as writing to the container's stdout. That'd mean that writing directly to it in addition should cause double logging shouldn't it? Since we're not wiping out our console handler right?

Reply:
This works because we're checking the parent pid. For dumb-init spawned services, the ppid will always be 1. This line then catches anyone doing a shell/script outside of the PID1 process tree.

Reply:
Ahhh I totally missed that second "p" in `getppid`.  Thanks.

So if we're running under dumb-init, but we forgot to put `exec` in `container-run.sh`, we'd be a child of a bash process that itself is a child of dumb-init, right? Do we need to walk the tree up? 

Maybe to improve readability it'd be useful to extract the ppid into a nicely named variable so it's obvious what's going on? e.g. `parent_pid = os.getppid()`.  (Or maybe I'm just the only one that can't spell haha)